e2e/cypress/test_module/cypress/support/commands.ts

```diff
@@ -44,9 +44,22 @@ Cypress.Commands.add("getProductsWithQuantity", (quantity) => {
 });
 
 Cypress.Commands.add("getProductsServerError", (quantity) => {
-  cy.intercept(httpMethod.get,  new RegExp(`\/api\/v1\/products\\?page=0&size=${quantity}(?:&.*)?`), {
-    statusCode: httpStatusCode.internalServerError
-  }).as("getProductsRequestServerError");
+  cy.intercept(
+    httpMethod.get,
+    new RegExp(`\/api\/v1\/products\\?page=0&size=${quantity}(?:&.*)?`),
+    {
+      statusCode: httpStatusCode.internalServerError
+    }
+  ).as("getProductsRequestServerError");
+});
+
+Cypress.Commands.add("signIn", () => {
+  cy.visit("/");
```

Comment:
The command for login with role was implemented before.
You should reuse it
![image](https://github.com/user-attachments/assets/be2103d7-bf62-470a-ba51-c0982b17e65d)

Reply:
Reuse `I authenticate to the system under role {word}` step

Reply:
Used existing. Thank you